scripts/webpack.config.js

```diff
@@ -99,6 +97,13 @@ function getConfig(isReactExternalized) {
                 },
             ],
         },
+        optimization: {
+            minimizer: [
+                // For webpack@5 you can use the `...` syntax to extend existing minimizers (i.e. `terser-webpack-plugin`), uncomment the next line
+                // `...`,
```

Comment:
i don't quite understand this comment. when do we uncomment this line?

Reply:
removed